cypress/integration/project_creation.spec.js

```diff
@@ -60,7 +60,6 @@ const withPlatinumStub = callback => {
 beforeEach(() => {
   cy.nukeAllState();
   cy.createIdentity();
-  cy.createProjectWithFixture();
```

Comment:
This was here because proxy would error out if we didn't have a project after nuking all state previously. It is now fixed and this workaround is not needed anymore.